docs/extensions/WebView.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: WebViewExtensions
+title: WebView extension
```

Comment:
should remain plural as there is more than one extension option, even though it's a single class